spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowTaskExecutionMetadataDao.java

```diff
@@ -118,7 +119,8 @@ public void save(TaskExecution taskExecution, TaskManifest manifest) {
 	@Override
 	public TaskManifest getLatestManifest(String taskName) {
 		Map<String, Order> sortKeys = new HashMap<>(1);
-		sortKeys.put("e.TASK_EXECUTION_ID", Order.DESCENDING);
+		// Make sure to add the sort keys into
+		sortKeys.put(SqlPagingQueryUtils.getSupportedKey("TASK_EXECUTION_ID"), Order.DESCENDING);
```

Comment:
I don't quite understand how this would cause any changes?

Reply:
Currently, the **only** place I see the sort parameter being used is at this code and hence added this to be part of the validation of sort keys.